google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineStandardFacetType.java

```diff
@@ -56,12 +56,6 @@ public boolean isSuitableModuleType(ModuleType moduleType) {
     return moduleType instanceof JavaModuleType;
   }
 
-  @NotNull
-  @Override
-  public String getDefaultFacetName() {
```

Comment:
Don't know why these changes are still showing up here, when I've already submitted them in other PRs, rebased both branches with origin/master and pushed.